applications_tests/lethe-fluid/dg_tracer_pipe.output

```diff
@@ -17,65 +17,257 @@ Tracer flow rate at the boundaries:
+	     Min: -0.0105187
+	     Max: 0.99387
+	 Average: 0.479834
+	 Std-Dev: 0.330862
+------------------
+Tracer flow rates
+------------------
+Tracer flow rate at the boundaries: 
+	 boundary 0: 0
+	 boundary 1: 0.0286677
+	 boundary 2: -0.0215134
+
+*******************************************************************************
+Transient iteration: 11       Time: 0.275    Time step: 0.025    CFL: 1.60881 
+*******************************************************************************
+Tracer statistics: 
+	     Min: -0.0150248
+	     Max: 0.993885
+	 Average: 0.491878
+	 Std-Dev: 0.328848
+------------------
+Tracer flow rates
+------------------
+Tracer flow rate at the boundaries: 
+	 boundary 0: 0
+	 boundary 1: 0.0286566
+	 boundary 2: -0.0232047
+
+*******************************************************************************
+Transient iteration: 12       Time: 0.3      Time step: 0.025    CFL: 1.60881 
+*******************************************************************************
+Tracer statistics: 
+	     Min: -0.0200827
+	     Max: 0.993842
+	 Average: 0.500791
+	 Std-Dev: 0.326659
+------------------
+Tracer flow rates
+------------------
+Tracer flow rate at the boundaries: 
+	 boundary 0: 0
+	 boundary 1: 0.0286472
+	 boundary 2: -0.0244888
+
+*******************************************************************************
+Transient iteration: 13       Time: 0.325    Time step: 0.025    CFL: 1.60881 
+*******************************************************************************
+Tracer statistics: 
+	     Min: -0.0254128
+	     Max: 0.993842
+	 Average: 0.507393
+	 Std-Dev: 0.324501
+------------------
+Tracer flow rates
+------------------
+Tracer flow rate at the boundaries: 
+	 boundary 0: 0
+	 boundary 1: 0.0286393
+	 boundary 2: -0.0254425
+
+*******************************************************************************
+Transient iteration: 14       Time: 0.35     Time step: 0.025    CFL: 1.60881 
+*******************************************************************************
+Tracer statistics: 
+	     Min: -0.0307408
+	     Max: 0.993821
+	 Average: 0.512326
+	 Std-Dev: 0.322416
+------------------
+Tracer flow rates
+------------------
+Tracer flow rate at the boundaries: 
+	 boundary 0: 0
+	 boundary 1: 0.0286329
+	 boundary 2: -0.0261338
+
+*******************************************************************************
+Transient iteration: 15       Time: 0.375    Time step: 0.025    CFL: 1.60881 
+*******************************************************************************
+Tracer statistics: 
+	     Min: -0.0358386
+	     Max: 0.993802
+	 Average: 0.516082
+	 Std-Dev: 0.320379
+------------------
+Tracer flow rates
+------------------
+Tracer flow rate at the boundaries: 
+	 boundary 0: 0
+	 boundary 1: 0.0286279
+	 boundary 2: -0.0266201
+
+*******************************************************************************
+Transient iteration: 16       Time: 0.4      Time step: 0.025    CFL: 1.60881 
```

Comment:
Is there a reason for simulating until 0.4s? Does that lead to steady-state or something in particular?